### ISSUE TITLE
Uunexpected err=NoSuchElementException(), type(err)=<class 'selenium.common.exceptions.NoSuchElementException'>

### ISSUE THREAD

**Issue by BIPTOA** [charlesbel/Microsoft-Rewards-Farmer#152]
I used fork by @AlexyGalaxy, but I keep getting this error. I think it might be error with trying to find elements before they load in. I tried increasing some wait values but it didn't help 
←[91m
[ERROR] An Error has Occured While Trying to Wait Until Visible.
←[00m
←[91m
[ERROR] Unexpected err=NoSuchElementException(), type(err)=<class 'selenium.common.exceptions.NoSuchElementException'>

**Comment by AlexyGalaxy** [comment]
Can you post a screenshot of your cmd/console using the snipping tool? When is this error occurring?

**Comment by BIPTOA** [comment]
@AlexyGalaxy 
Sorry I forgot to attach it . Also I'm from EU I know there used to be something weird with that. 
![image](https://user-images.githubusercontent.com/93800136/217410424-9269273e-53e9-4f8a-9a7a-5b63ba8c6659.png)

**Comment by AlexyGalaxy** [comment]
Have you ran this script before or is this your first run?

Find line 1135 and comment it out by placing a # in front of it. I will test this tomorrow.


```
#waitUntilVisible(browser, By.XPATH, '//*[@id="daily-sets"]/mee-card-group[1]/div/mee-card[1]/div/card-content/mee-rewards-daily-set-item-content/div', 10)
```

**Comment by BIPTOA** [comment]
I did comment out it ![image](https://user-images.githubusercontent.com/93800136/217515172-cd08b9a3-3ea3-4851-9668-b634e78c0176.png)
Still the same error
![image](https://user-images.githubusercontent.com/93800136/217515049-e886858e-da6d-41c6-8be0-2e21478829a1.png)
This is my first time trying to run this version of scripts. I used to use one from someone else.

**Comment by AlexyGalaxy** [comment]
Have you ever successfully ran the script or is this your first run? (This current version does work for me) You might have not installed something correctly or your browser is taking longer than expected to load and the script is not waiting long enough.

First, disable headless mode by commenting out line 46 and 47 with #. Then watch the script open chrome and look to see where it is stuck. (When done testing make sure to delete the # to rentable headless mode.
```
#if headless_mode : #comment out to disable headless mode (makes window visable) 
           #options.add_argument("--headless") #comment out to disable headless mode (makes window visable) 
```

Pause the scrip by placing the code below on line 1122, before the waitUntilVisible and once again after the waitUntilVisible. Then run the script and post the screen shot here. If the browser is taking too long to load, increase the number at the end of waitUntilVisible to 30-50(this is the time waited)

```
print('[TEST] Press any key to continue')
input()
```

**Comment by BIPTOA** [comment]
Okay I think I managed to fix it. I think this script was just too good, for version of rewards we have in Poland. I disabled couple of things in the script, and changed url (it's a little diffrent on EU) in one place. Thank you for pointing me in right direction. For some reason it loops now twice at the end, but script works. So now I just need to run this script once per day and I'm set. Thank you very much.
![image](https://user-images.githubusercontent.com/93800136/217559231-bfbb92d6-0de4-4dd0-bd0e-b73982e64db6.png)

[ms_rewards_farmer.txt](https://github.com/charlesbel/Microsoft-Rewards-Farmer/files/10686555/ms_rewards_farmer.txt)

**Comment by AlexyGalaxy** [comment]
Do you mind posting the changes and which lines they were on just in case someone else has this issue? It could help others stuck on the same issue. Thanks

**Comment by BIPTOA** [comment]
> Do you mind posting the changes and which lines they were on just in case someone else has this issue? It could help others stuck on the same issue. Thanks

Sorry I didn't see your reply, here is new version working. It allows for mobile searches also, it prints some errors at the end, but it works, when I try to remove them it decides to stop working... 
[ms_rewards_farmer_EU.txt](https://github.com/charlesbel/Microsoft-Rewards-Farmer/files/11157937/ms_rewards_farmer_EU.txt)

**Comment by charlesbel** [comment]
Fixed with v3.0 #188
